app/assets/javascripts/scp-related-genes-ideogram.js

```diff
@@ -13,8 +13,21 @@
 
 /** Handle clicks on Ideogram annotations */
 function onClickAnnot(annot) {
+  const ideo = this // eslint-disable-line
```

Comment:
what is the 'this' here?  is it the ideoConfig object?

Reply:
It's the Ideogram object (the config object is accessible via `ideogram.config`, FWIW).  I renamed the variable for clarity and added a brief comment in dbd7beb.